Xbim.IO.Table/Xbim.IO.Table.csproj

```diff
@@ -4,11 +4,13 @@
     <AssemblyName>Xbim.IO.Table</AssemblyName>
     <RootNamespace>Xbim.IO.Table</RootNamespace>
     <Title>Xbim Table Store</Title>
-    <Description>Provides support for spreadsheet serialization/deserialization of data stored in xBIM model generated from EXPRESS schema</Description>
+		<LangVersion>8.0</LangVersion>
+		<Description>Provides support for spreadsheet serialization/deserialization of data stored in xBIM model generated from EXPRESS schema</Description>
     <PackageIcon>Icon.png</PackageIcon>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NPOI" Version="2.6.2" />
+		<PackageReference Include="NPOI" Version="2.6.2" />
```

Comment:
Can we drop the reference to NPOI now? Or are we still using it?

Reply:
there is one static colors used for styling the columns but we are not using any other reading or writes from NPOI